addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -22,22 +22,38 @@
                         </div>
                     </div>
                 </section>
-                <div t-if="event.is_finished" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is finished. It's no longer possible to book a booth.</span>
+                <div t-if="event.is_finished">
+                    <div class="col-12">
+                        <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                            <div class="mt-5 mb-3">
+                                <t t-call="website_event.event_svg"/>
+                            </div>
+                            <h2 class="mb-2">Event Finished</h2>
+                            <p>It's no longer possible to book a booth.</p>
+                        </div>
+                    </div>
                 </div>
-                <div t-elif="not event_booths" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is not open to exhibitors registration,
-                        <span t-if="request.env.user.has_group('event.group_event_manager')">
-                            you can
-                            <a class="text-nowrap" t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
-                                <i class="fa fa-gear me-1" role="img" aria-label="Configure" title="Configure event booths"/><em>Configure Booths</em>
-                            </a>
-                            for this event.
-                        </span>
-                        <span t-else="">
-                            check our <a href="/event">list of future events</a>.
-                        </span>
-                    </span>
+                <div t-elif="not event_booths">
+                    <div class="col-12">
+                        <div class="d-flex flex-column align-items-center justify-content-center text-center">
+                            <div class="mt-5 mb-3">
+                                <t t-call="website_event.event_svg"/>
+                            </div>
+                            <h2 class="mb-2">Registration Not Open</h2>
+                            <p>This event is not open to exhibitors registration at this time</p>
+                        </div>
+                        <div class="o_not_editable d-flex flex-column align-items-center justify-content-between my-3" groups="event.group_event_user">
+                            <div t-if="request.env.user.has_group('event.group_event_manager')" class="d-flex flex-column align-items-center justify-content-between">
```

Comment:
Is there any reason we don't use ``groups`` here?

Reply:
I'm not sure to understand here sorry

Reply:
No problem I'm asking whether we could do this, or if there's a reason to use ``has_group``
```suggestion
                            <div groups="event.group_event_manager" class="d-flex flex-column align-items-center justify-content-between">
```

Reply:
Not yet familiar with that. It was already set like that. Should I change with your suggestion?

Reply:
Hm, well if it was already like that I guess that's fine. I'm genuinely not sure if there could have been a reason, so let's leave it like that ^^